src/constants/walletconnect.js

```diff
@@ -0,0 +1,88 @@
+export const PLUG_DESCRIPTION = {
```

Comment:
all of this is temporary right? otherwise you should extract the messages into the translation file

Reply:
I think we should put only description attribute into translation file, the others attributes are the same in any language